package.json

```diff
@@ -16,8 +16,8 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
-    "@aeternity/aepp-components": "2.0.15",
-    "@aeternity/aepp-components-3": "3.0.0-alpha.4",
+    "@aeternity/aepp-components": "2.0.14",
```

Comment:
Don't downgrade version of `aepp-components` it breaks handling of `ak_` addresses in AeAddressInput.